src/fragments/lib-v1/storage/js/download.mdx

```diff
@@ -71,10 +73,17 @@ Download a file to in-memory buffer.
 import { downloadData } from 'aws-amplify/storage';
 
 // Download a file from s3 bucket
-const { body, eTag } = await downloadData({ key, data: file, options: {
-onProgress, // Optional progress callback.
-} }).result;
-```
+const { body, eTag } = await downloadData({
+	key,
+	data: file,
+	options: {
```

Comment:
Up to here this seems like it's an executable call, but from here on it looks like typing?